src/main/java/org/mastodon/mamut/tomancak/TomancakPlugins.java

```diff
@@ -238,11 +244,12 @@ public List< ViewMenuBuilder.MenuItem > getMenuItems()
 								item( FLIP_DESCENDANTS ),
 								item( SORT_TREE ),
 								item( SORT_TREE_EXTERN_INTERN ),
-								item( LABEL_SPOTS_SYSTEMATICALLY ) ),
+								item( LABEL_SPOTS_SYSTEMATICALLY ),
+								item( MIRROR_SPOTS ) ),
```

Comment:
As a user who is non familiar with this set of plugins, I would not expect this functionality under the category "trees management", as there is not directly something happening with the tree itself. Since, I am not aware of the already existing categories, I am not sure, if it would match some other category that already exists or if a new category, e.g. "Spot Operations" would make sense.

Reply:
changed as suggested.